src/electron/device-connect-view/components/window-footer.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { css } from '@uifabric/utilities';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import * as React from 'react';
+import { NamedSFC } from '../../../common/react/named-sfc';
+
+export interface WindowFooterProps {
+    cancelClick: () => void;
+}
+
+export const WindowFooter = NamedSFC<WindowFooterProps>('WindowFooter', (props: WindowFooterProps) => {
+    const onCancelClick = () => props.cancelClick();
+    return (
+        <footer className="ms-Grid window-footer">
```

Comment:
Is it necessary for this component to use the `ms-Grid`/`ms-Grid-row` classes? My understanding is that we usually try to treat the `ms-*` classes as office fabric implementation details; I think ideally we'd avoid using their classes unless there was some very compelling reason for it

Reply:
(same throughout for other components)

Reply:
Nope, not necessary. Removed.